modules/core/src/main/resources/artifact-migrations.conf

```diff
@@ -208,5 +208,10 @@ changes = [
     artifactIdBefore = argonaut-refined_6.2
     artifactIdAfter = argonaut-refined_6.3
     initialVersion = 1.3.0
+  },
+  {
+    groupIdAfter = org.polyvariant
```

Comment:
@kubukoz The `groupIdBefore` is missing. Too bad this does not result in parse / test error.

Reply:
Thanks, I'll fix it later (need to figure out why publishing doesn't work first)